### ISSUE TITLE
Please update the npm package

### ISSUE THREAD

**Issue by Kreijstal** [rustwasm/wasm-pack#1376]
## 🐛 Bug description
Describe your issue in detail
```
npm ERR! npm ERR! C:\Users\topkek\AppData\Local\npm-cache\_cacache\tmp\git-cloneXXXXXXh89pxf\node_modules\wasm-pack\binary.js:23
npm ERR! npm ERR!   throw new Error(`Unsupported platform: ${type} ${arch}`);
npm ERR! npm ERR!   ^
npm ERR! npm ERR!
npm ERR! npm ERR! Error: Unsupported platform: MINGW32_NT-10.0 x64
npm ERR! npm ERR!     at getPlatform (C:\Users\topkek\AppData\Local\npm-cache\_cacache\tmp\git-cloneXXXXXXh89pxf\node_modules\wasm-pack\binary.js:23:9)
npm ERR! npm ERR!     at getBinary (C:\Users\topkek\AppData\Local\npm-cache\_cacache\tmp\git-cloneXXXXXXh89pxf\node_modules\wasm-pack\binary.js:27:20)
npm ERR! npm ERR!     at install (C:\Users\topkek\AppData\Local\npm-cache\_cacache\tmp\git-cloneXXXXXXh89pxf\node_modules\wasm-pack\binary.js:36:18)
npm ERR! npm ERR!     at Object.<anonymous> (C:\Users\topkek\AppData\Local\npm-cache\_cacache\tmp\git-cloneXXXXXXh89pxf\node_modules\wasm-pack\install.js:4:1)
npm ERR! npm ERR!     at Module._compile (node:internal/modules/cjs/loader:1368:14)
npm ERR! npm ERR!     at Module._extensions..js (node:internal/modules/cjs/loader:1426:10)
npm ERR! npm ERR!     at Module.load (node:internal/modules/cjs/loader:1205:32)
npm ERR! npm ERR!     at Module._load (node:internal/modules/cjs/loader:1021:12)
npm ERR! npm ERR!     at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:142:12)
npm ERR! npm ERR!     at node:internal/main/run_main_module:28:49
```
.

#### 🤔 Expected Behavior
What should have happened?
You have a MSVC build for windows, can you do a mingw one as well? use the rust target `x86_64-pc-windows-gnu`

#### 👟 Steps to reproduce
Clear steps describing how to reproduce the issue, including commands and flags run. If you are seeing an error, please include the full error message and stack trace.

go to msys2/mingw64
install npm
`npm install wasm-pack` 
get error

#### 🌍 Your environment
Include the relevant details of your environment.
wasm-pack version: master branch
rustc version:rustc 1.76.0 (07dca489a 2024-02-04) (Rev1, Built by MSYS2 project)

**Comment by Kreijstal** [comment]
Eh I just saw there was a pull request added recently, is it not published on the npm registry yet?

**Comment by drager** [comment]
Released in [v0.13.0](https://www.npmjs.com/package/wasm-pack/v/0.13.0).
